### ISSUE TITLE
Module name colliding with other packages

### ISSUE THREAD

**Issue by alexisrolland** [ivelum/graphql-py#12]
The module name `graphql` when doing the import is very generic and collides with other packages I'm using. If you plan some evolutions in the future, I would suggest to change it to something more specific such as `graphqlparser` or similar...

Thanks for this good package!

**Comment by stebunovd** [comment]
hey Alexis,

could you please clarify, with which exactly other packages does it collide? From what I see it doesn't collide with anything from here https://graphql.org/code/#python-1

thanks,

Denis

**Comment by alexisrolland** [comment]
Hi Denis,
I was under the impression your package name collides with https://graphene-python.org because after installing `graphene` I can do `import graphql` and after installing `graphql-py` I can also do `import graphql`. Can you please help me to understand what happens in the background? Are both packages using in a shared `graphql` folder / package?
Thanks

**Comment by stebunovd** [comment]
yeah, it turns out graphene depends on graphql-core, which introduces `graphql` package. Ok, thanks for the note. I'll consider renaming in future releases
